docs/docs/sample-scripts/spo/list-tenant-alert-usage/index.mdx

```diff
@@ -0,0 +1,108 @@
+---
+tags:
+  - reports
+  - alerts
+  - retirement
+  - tenant
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# List SharePoint alerts usage across the tenant
+
+Author: [Saurabh Tripathi](https://github.com/saurabh7019)
+
+SharePoint Online list alerts are being gradually retired. This script scans all sites across the tenant and generates a comprehensive CSV report of existing alerts. This information helps administrators identify and plan the migration of critical alerts to modern alternatives.
+
+<Tabs>
+  <TabItem value="PowerShell">
+
+  ```powershell
+  $fileExportPath = "Alerts.csv"
+
+  try {
+    $m365Status = m365 status --output text
+
+    if ($m365Status -eq "Logged Out") {
+      # Connection to Microsoft 365
+      m365 login
+    }
```

Comment:
Instead of this, we could use `login --ensure`. This will only trigger a login flow when there is no active connection.